apps/src/StudioApp.js

```diff
@@ -1612,7 +1613,7 @@ StudioApp.prototype.builderForm_ = function (onAttemptCallback) {
 */
 StudioApp.prototype.report = function (options) {
   // copy from options: app, level, result, testResult, program, onComplete
-  var report = $.extend({}, options, {
+  var report = Object.assign({}, options, {
```

Comment:
Since including it at the top of code-studio.js, [which gets loaded here](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/layouts/application.html.haml#L13) this should work everywhere except for really unusual places like embedded blocks.